src/AddressRegistry.Api.Oslo/AddressMatch/Responses/AddressMatchCollection.cs

```diff
@@ -149,11 +161,11 @@ public class AdresMatchItem
 
         public static AdresMatchItem Create(
             AdresMatchScorableItem scorableItem,
-            AddressMatchContext addressMatchContext,
             ResponseOptions responseOptions)
         {
             return new AdresMatchItem
             {
+                Context = responseOptions.ContextUrlDetail,
```

Comment:
Should be it's own context. `ContextUrlAdresmatch` ? Also change appsettings.

Reply:
Added ContextUrlAddressMatch